src/deadline.json

```
@@ -14,6 +14,13 @@
       "day": 13,
       "hour": 9
     },
+    {
+      "name": "卒業",
+      "year": 2024,
+      "month": 4,
+      "day": 1,
+      "hour": 0
```

```suggestion
      "month": 3,
      "day": 25,
      "hour": 12
```
どうかな
https://www.tsukuba.ac.jp/campuslife/calendar-ceremony/graduation/